cli/src/main/scala/aqua/AppOpts.scala

```diff
@@ -86,7 +86,7 @@ object AppOpts {
             if (exists && isDir) Validated.validNec[String, Path](p)
             else
               Validated.invalidNec[String, Path](
-                s"There is no path ${p.toString} or it is not a directory"
+                s"'${p.toString}' path does not lead to a directory"
```

Comment:
what does 'lead to directory' mean? I've never seen such terminology, and I guess most our users didn't, too. The previous error was rather standard.